packages/syft/src/syft/service/worker/utils.py

```diff
@@ -177,7 +177,7 @@ def run_container_using_docker(
             image=worker_image.image_identifier.full_name_with_tag,
             name=f"{hostname}-{worker_name}",
             detach=True,
-            auto_remove=True,
+            # auto_remove=True,
```

Comment:
How come we have commented this out?

Reply:
yeah this was to detect the errors in default worker pool, need to clean up this PR before we merge